lib/Makefile.include

```diff
@@ -23,7 +23,7 @@ ifneq ($(V),1)
 Q := @
 endif
 
-TOPDIR = $(SRCLIBDIR)/..
+TOPDIR := $(SRCLIBDIR)/..
```

Comment:
this is changing the line you added in the previous commit. One of them is wrong :)